guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -139,6 +139,23 @@ gpgcheck=1
 ----
 endif::[]
 
+ifdef::satellite[]
+. If there are multiple network interfaces, fix the interface names and rerun the installer with the new interface name.
+Call the installer to find out what interface was used and where:
++
+----
+# satellite-installer --help |grep 'interface.*eth'
```

Comment:
Please use attributes instead. You'll also need to allow attributes in the code block.